.env.example

```
@@ -44,3 +44,32 @@ AWS_ACCESS_KEY_ID=
 AWS_SECRET_ACCESS_KEY=
 AWS_DEFAULT_REGION=ap-south-1
 AWS_S3_BUCKET_PREFIX="bucket-prefix-name"
+
+# RabbitMQ Configuration (Celery Broker)
+RABBITMQ_HOST=localhost
+RABBITMQ_PORT=5672
+RABBITMQ_USER=guest
+RABBITMQ_PASSWORD=guest
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The RABBITMQ_PASSWORD key should go before the RABBITMQ_PORT key

(UnorderedKey)